backend/handlers/handlers_test.go

```diff
@@ -57,19 +98,179 @@ func TestListRoutes(t *testing.T) {
 		if len(methods) > 0 {
 			fmt.Print(" (" + strings.Join(methods, ",") + ")")
 		}
-		/* The following methods are only available in a newer version of mux: */
-		// queriesTemplates, err := route.GetQueriesTemplates()
-		// if err == nil {
-		// 	   fmt.Println("Queries templates:", strings.Join(queriesTemplates, ","))
-		// }
-		// queriesRegexps, err := route.GetQueriesRegexp()
-		// if err == nil {
-		// 	   fmt.Println("Queries regexps:", strings.Join(queriesRegexps, ","))
-		// }
 		fmt.Println()
 		return nil
 	})
 	if err != nil {
 		fmt.Println(err)
 	}
 }
+
+func requestTester(request *http.Request, t *testing.T) *httptest.ResponseRecorder {
+	connectionData := handlers.NewConnectionData(8082, "")
+	backendInstance, err := backend.NewBackend(arguments.NewArguments(
+		test.TstTempDir("test-tempdir-"), false, false, false, false, false),
+		webdevEnvironment{},
+	)
+	if err != nil {
+		t.Errorf("An error occurred. %v", err)
+	}
+
+	handlers := handlers.NewHandlers(backendInstance, connectionData)
+
+	rr := httptest.NewRecorder()
+	handlers.Router.ServeHTTP(rr, request)
+	return rr
+}
+
+func checkResponse(t *testing.T, response *httptest.ResponseRecorder, expectedResponse string) {
+	if status := response.Code; status != http.StatusOK {
+		t.Errorf("Status code differs. Expected %d. Got %d", http.StatusOK, status)
+	}
+	if response.Body.String() != expectedResponse {
```

Comment:
same here, `require.Equal(response.Body.String(), expectedResponse)`

(and above and everywhere else ;))